testing-unit/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+apply from: '../gradle/publication.gradle'
+
+android {
+    namespace 'com.bumble.appyx.testing'
+    compileSdk 32
+
+    defaultConfig {
+        minSdk 21
+        targetSdk 32
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8
+        useIR = true
+    }
+    lintOptions {
+        abortOnError false
+    }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion libs.versions.compose.get()
+    }
+}
+
+dependencies {
+    api project(':core')
+    api libs.androidx.test.rules
+
+    implementation libs.androidx.lifecycle.java8
+    implementation libs.androidx.lifecycle.runtime
+    implementation libs.compose.ui.ui
+    implementation libs.junit
+    implementation libs.kotlin.coroutines.test
+    implementation libs.mvicore
```

Comment:
I think this is okay for this PR, however I think we should extract the MVICore and RIBs test classes into different test modules.

i.e. 
`testing-unit-core`
`testing-unit-mvicore`
`testing-unit-ribs`

Otherwise the testing dependency will pull in APIs they don't necessarily want to use.

I'm happy to pick this up after this gets merged if you like?

Reply:
I removed these dependencies from `testing-unit`

Reply:
thanks